documentation/2.0/content/userguide/tools/model_help.md

```diff
@@ -114,17 +114,17 @@ Recursive sub-folders only for resources:/JDBCSystemResource
 
 resources:
     JDBCSystemResource:
-        'JDBC-1':
+        JDBC-1:
             JdbcResource:
                 JDBCConnectionPoolParams:
                 JDBCDataSourceParams:
                 JDBCDriverParams:
                     Properties:
-                        'Properties-1'
+                        Properties-1:
                 JDBCOracleParams:
                 JDBCXAParams:
             SubDeployment:
-                'SubDeployment-1':
+                SubDeployment-1:
```

Comment:
These changes need to be reverted - this is showing the output from modelHelp, which still has the unnecessary quoting.
The model example in this doc is already handled in #1064